docs/QuickStart.md

```diff
@@ -52,7 +52,7 @@ These steps can be done in the Terminal/CLI or with PowerShell.
 
 1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
```

Comment:
Remove the language-specific setup now that it's in each separate guide. This one should only contain general setup

(and it wouldn't hurt to include a link to this in each of those guides as well to make sure someone doesn't go directly there and miss doing these)